llama_stack/apis/tools/tools.py

```diff
@@ -57,18 +57,35 @@ class ToolDef(BaseModel):
     )
 
 
+@json_schema_type
+class MCPInlineConfig(BaseModel):
+    type: Literal["inline"] = "inline"
+    command: str
```

Comment:
I don't think we should do this. This is asking for trouble where a server is accepting an arbitrary command to run without any promise of isolation, etc. The MCP model works well with llama stack _only_ when the user has figured out how to run a server. If they cannot do that, we just put up our hands.